### ISSUE TITLE
After upgrade to json5@2, rollup bundle fails

### ISSUE THREAD

**Issue by plantain-00** [json5/json5#182]
### demo index.js

```js
import JSON5 from 'json5'

console.info(JSON5.stringify('aa'))
```

### demo rollup.config.js

```js
import { uglify } from 'rollup-plugin-uglify'
import resolve from 'rollup-plugin-node-resolve'
import commonjs from 'rollup-plugin-commonjs'

export default {
  input: 'index.js',
  name: 'Test',
  plugins: [
    resolve(),
    uglify(),
    commonjs()
  ],
  output: {
    file: 'index.min.js',
    format: 'umd'
  }
}
```

`rollup --config rollup.config.js`

### error message

```txt
index.js → index.min.js...
  17 |
  18 |     var util = {
> 19 |         isSpaceSeparator (c) {
     |                         ^ Unexpected token: punc (()
  20 |             return unicode.Space_Separator.test(c)
  21 |         },
  22 |
[!] (uglify plugin) Error: Error transforming bundle with 'uglify' plugin: Unexpected token: punc (()
Error: Error transforming bundle with 'uglify' plugin: Unexpected token: punc (()
    at error (/Users/yaoyao/.config/yarn/global/node_modules/rollup/dist/rollup.js:3365:15)
    at /Users/yaoyao/.config/yarn/global/node_modules/rollup/dist/rollup.js:13868:17
    at <anonymous>
```

It seems uglify-js is able to transpile only es5 syntax, and I'm using json5 in browser environment and target es5.

I'm not sure this is json5's bug(for example if json5 does not target es5 anymore, it is not a bug), feel free to close if not.

**Comment by jordanbtucker** [comment]
json5@2.0.0 no longer targets ES5, and instead it targets Node.js v6 or greater. Moving forward, new major versions of json5 will target the oldest LTS version of Node.js at the time of publishing.

If you're using Node.js v6+, then you probably shouldn't be using rollup-plugin-uglify (or uglify-js) since it doesn't support ES6+. I recommend switching from rollup-plugin-uglify to rollup-plugin-terser, which does support ES6+. If you're using Node.js v5 or lower, then you should stick with json5@1.0.1.

Alternatively, you can use the bundle at `dist/index.js` in v2.0.0, which still targets ES5 and works in browser and Node.js environments.

```js
import JSON5 from 'json5/dist/index'

console.info(JSON5.stringify('aa'))
```

**Comment by plantain-00** [comment]
I just tried to import from 'json5/dist/index', it still not target ES5, so not work in browser environment.

**Comment by jordanbtucker** [comment]
Oh, sorry. You're right. The bundle at `dist/index.js` is just bundled with roll-up. The bundle at `dist/index.min.js` is bundled with rollup-plugin-terser, so it should work be ES5. I'll fix this in 2.0.1.

**Comment by jordanbtucker** [comment]
Actually, I just realized that rollup-plugin-terser doesn't output ES5 by default either. So json5@2.0.0 has no support for ES5. I'll get that fixed in 2.0.1.

**Comment by jordanbtucker** [comment]
Fixed in [v2.0.1](https://github.com/json5/json5/releases/tag/v2.0.1).
